server.js

```diff
@@ -3,6 +3,7 @@ const fetch = require('node-fetch');
 require('dotenv').config();
 const {configuration, OpenAI} = require('openai');
 const app = express();
+let country = "";
 
```

Comment:
Adding the 'Country' global variable was a good move. This seems to have fixed issue we were having with GPT not being able to react to the selected country response.